src/components/PageHeader.vue

```diff
@@ -1,5 +1,28 @@
 <template>
   <header>
+    <div class="flex justify-between items-center mt-10 pb-8 border-b-2 mx-4">
+      <Transition>
+        <ConfirmationModal
+          v-if="isPopUpShown"
+          title="Warning"
+          text="Unsuccesfull logout"
+          @hide-pop-up="hidePopUp"
+        />
+      </Transition>
+      <div class="flex flex-col text-black lg:text-lg font-medium">
+        <p>Welcome</p>
+        <p>{{ `${user.firstName} ${user.lastName}` }}</p>
```

Comment:
If you change this string concatenation to `{{ user.firstname }} {{ user.lastName }}` the result will be the same.
Or you can create a computed in the script and concatenate this there:
`{{ userName}}` in template and `const userName = computed(() => `${user.firstName} ${user.lastName}`); ` in the script.
Both of them are correct, but you version is a bit redundant since the {{ }} evaluates the variables as well.